packages/search-types/src/request/request.model.ts

```diff
@@ -52,5 +52,7 @@ export interface TrackableRequest {
  * @public
  */
 export interface ExtraParamsRequest {
-  [key: string]: any;
+  extraParams: {
+    [key: string]: any;
```

Comment:
I think @javieri-empathy suggested to revert this `any` to `unknown`, but he can correct me if I'm wrong.

Reply:
changed